dockerfiles/e2e/entrypoint.sh

```diff
@@ -12,6 +12,19 @@ if [ -z "$TEST_SUITE" ]; then
     TEST_SUITE=test-happy-path
 fi
 
+# Launch display mode and VNC server
+export DISPLAY=':20'
+Xvfb :20 -screen 0 1920x1080x16 > /dev/null 2>&1 &
+x11vnc -passwd 123456 -display :20 -N -forever > /dev/null 2>&1 &
```

Comment:
Is it required to set up a password? Wouldn't it be better without it?

Reply:
Thank you, password removed.